File: dayFour/java/Application.java
public class Application{
 
public static int remainingTransaction(int initialDeposit, int deposit, int withdraw){
int depositBalance = deposit + initialDeposit;
int withdrawBalance = depositBalance - withdraw;

if (withdraw > depositBalance) {
System.out.println("withdrawal failed: insufficient funds");
return depositBalance; 

} 

else {

return withdrawBalance;
}
}
}
       
      
      

File: dayFour/python/Application.py
class Application:
	def the_app(self,initial_deposit,deposit,withdraw)
		deposit_balance = deposit - initial_deposit 
        withdraw_balance = deposit_balance - withdraw
        if (withdraw > deposit_balance):
			print("withdrawal failed: insufficient funds")
			return deposit_balance
			else:
			return withdrawBalance

       
     
